test/files/setup_databases.jl

```diff
@@ -35,24 +37,32 @@ function parse_commandline()
         "--output", "-o"
         help = "path where the database folder is going to be created."
         default = "."
-        "--pdb"
+        """
+–pdb
+"""
```

Comment:
this formatting seems quite a bit worse

Reply:
Oof yeah that is worse. This isn't even a docstring is it?

Reply:
It's parsed that way by CSTParser - I'm not sure if that's correct or not but we want to leave these strings alone, I think even doing the quote normalization thing isn't ideal in this case.

Reply:
Where does the code come from? I'm not sure what the `@add_arg_table` macro is doing. I tried parsing it in Julia and it does parse as a docstring...If they are intended to be markdown docstrings, `--pdb` should be wrapped in backticks